src/obproxy/obutils/ob_proxy_config.h

```diff
@@ -306,6 +306,9 @@ class ObProxyConfig : public common::ObCommonConfig
   // proxy cmd
   DEF_INT(proxy_local_cmd, "0", "[0,]", "proxy local cmd type: 0->none(default), 1->exit, 2->restart, 3->commit, 4->rollback", CFG_NO_NEED_REBOOT, CFG_SECTION_OBPROXY, CFG_VISIBLE_LEVEL_MEMORY);
 
+  DEF_BOOL(enable_weak_read, "false", "weak read by default for all clients connected this proxy", CFG_NO_NEED_REBOOT, CFG_SECTION_OBPROXY, CFG_VISIBLE_LEVEL_USER);
```

Comment:
We expect three classes of instances:
Read-write, automatic read-write separation, read-only

So here should be read-only configuration items, including weak read and write prohibited